src/axom/inlet/Table.cpp

```diff
@@ -51,8 +50,100 @@ std::shared_ptr<Table> Table::addTable(const std::string& name,
   return currTable;
 }
 
+std::shared_ptr<Table> Table::addBoolArray(const std::string& name, 
+                                           const std::string& description) {
+  auto table = addTable(appendPrefix(name,"_inlet_array"), description);
+  std::unordered_map<int,bool> map;
+  const std::string& fullName = appendPrefix(m_name, name);
+  if (m_reader->getBoolMap(fullName, map)) {
+    for (auto p : map) {
+      table->addBoolHelper(std::to_string(p.first), "", true, p.second);
+    }
+  } else {
+    SLIC_WARNING(fmt::format("Bool array {0} not found.", fullName));
+  }
+  return table;
+}
+
+std::shared_ptr<Table> Table::addIntArray(const std::string& name, 
+                                          const std::string& description) {
+  auto table = addTable(appendPrefix(name,"_inlet_array"), description);
+  std::unordered_map<int,int> map;
+  const std::string& fullName = appendPrefix(m_name, name);
+  if (m_reader->getIntMap(fullName, map)) {
+    for (auto p : map) {
+      table->addIntHelper(std::to_string(p.first), "", true, p.second);
+    }
+  } else {
+    SLIC_WARNING(fmt::format("Int array {0} not found.", fullName));
+  }
+  return table;
+}
+
+std::shared_ptr<Table> Table::addDoubleArray(const std::string& name, 
+                                             const std::string& description) {
+  auto table = addTable(appendPrefix(name,"_inlet_array"), description);
+  std::unordered_map<int,double> map;
+  const std::string& fullName = appendPrefix(m_name, name);
+  if (m_reader->getDoubleMap(fullName, map)) {
+    for (auto p : map) {
+      table->addDoubleHelper(std::to_string(p.first), "", true, p.second);
+    }
+  } else {
+    SLIC_WARNING(fmt::format("Double array {0} not found.", fullName));
+  }
+  return table;
+}
+
+std::shared_ptr<Table> Table::addStringArray(const std::string& name, 
+                                             const std::string& description) {
+  auto table = addTable(appendPrefix(name,"_inlet_array"), description);
+  std::unordered_map<int,std::string> map;
+  const std::string& fullName = appendPrefix(m_name, name);
+  if (m_reader->getStringMap(fullName, map)) {
+    for (auto p : map) {
+      table->addStringHelper(std::to_string(p.first), "", true, p.second);
+    }
+  } else {
+    SLIC_WARNING(fmt::format("String array {0} not found.", fullName));
+  }
+  return table;
+}
+
+bool Table::getBoolArray(std::unordered_map<int,bool>& map) {
+  size_t found = m_name.find("/_inlet_array");
```

Comment:
pretty sure that '/' is not supposed to be there.

Reply:
Is find specific enough here? is m_name a fully qualified name that may have one embedded in the full path? would endswith (there is axom::utilities::string::endsWith()) be better?